src/nft/components/view/NFTLastSoldPrice.tsx

```diff
@@ -0,0 +1,40 @@
+import { cn, text } from '../../../styles/theme';
+import { formatAmount } from '../../../token/utils/formatAmount';
+import { useNFTContext } from '../NFTProvider';
+
+type NFTLastSoldPriceProps = {
+  className?: string;
+  label?: string;
+};
+
+export function NFTLastSoldPrice({
+  className,
+  label = 'Mint price',
```

Comment:
should we standardize on `label`? we use `text` in our other components

Reply:
label seems more flexible if you ever want to change from string to ReactNode